Services/DataX.Config/DataX.Config.Test/FlowOperationTest.cs

```diff
@@ -27,11 +28,13 @@ public class FlowOperationTest
         public static void Initialize(TestContext tc)
         {
             InitialConfiguration.Set(Constants.ConfigSettingName_RuntimeKeyVaultName, "somekeyvault");
+            InitialConfiguration.Set(Constants.ConfigSettingName_SparkType, "hdinsight");
```

Comment:
is this optional?

Reply:
made it optional and removed it from hdInsights testcases. Added another end to end test case for databricks and setting sparktype in that test case